python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1726,9 +1726,10 @@ def start(
         if stream:
             return stream_stdout_and_stderr(full_cmd)
         elif attach:
-            return run(full_cmd)
+            run(full_cmd, tty=True)
+            return None
```

Comment:
To me, this is indeed a breaking change, but it makes sense. Furthermore, if a user is interested in stdout, he can either use stream or the normal call. For me this is acceptable (we're not in the v1.0 yet). Feel free to  adapt the unit tests. I'll add a big fat warning in the release notes. Also, can you add a warning in the docstring saying that the bahavior changed? Thanks!

Reply:
Sounds good, thanks!